src/main/kotlin/snyk/common/lsp/Types.kt

```diff
@@ -79,6 +85,185 @@ data class ScanIssue(
         endOffset = document?.getLineStartOffset(range.end.line)?.plus(range.end.character)
     }
 
+    fun type(): IssueType {
```

Comment:
Have a look at the new functions in this file especially to see if anything doesn't make sense. They're used throughout the UI so hopefully they should